umm-spec-lib/src/cmr/umm_spec/migration/version/interface.clj

```diff
@@ -0,0 +1,22 @@
+(ns cmr.umm-spec.migration.version.interface
+  "Contains the interface definition for migrating between versions of UMM
+  schemas.")
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; Interface for Migrating Between Versions
+
+(defn- dispatch-migrate
+  [_context _concept concept-type source-version dest-version]
+  [concept-type source-version dest-version])
+
+(defmulti migrate-umm-version
```

Comment:
I think it is easier to read if you put `dispatch-migrate` function inline. it is also the CMR convention. i.e.

```
(defmulti migrate-umm-version
  ""Returns the given data structure of the indicated concept type and UMM
  version updated to conform to the target UMM schema version."
  (fn [_context _concept concept-type source-version dest-version]
   [concept-type source-version dest-version]))
```

Reply:
Yeah, I can do that. I was a little surprised to see this broken out ... not the usual pattern in the CMR.

Done.